### ISSUE TITLE
Display animated gif

### ISSUE THREAD

**Issue by mejo-** [nextcloud/collectives#202]
In GitLab by @Hanskes on Jul 22, 2021, 09:39

We use a wordpress site next to our nextcloud for the company documentation. We would love to use collective for this purpose. However, there are a few things that we use a lot that I don't think work (yet) within collective. For example, we often use animated gif files to make certain actions transparent and we use interactive diagrams from draw.oi to show processes, which are placed as html fields in articles.

![planning](https://gitlab.com/collectivecloud/collectives/uploads/a42f6aa5033fa54b5da431f2822fb034/planning.png)

In other words, is it or will it be possible in the near future to insert html blocks and watch animated gif files directly?
